Common/API/Nodes.cs

```diff
@@ -0,0 +1,142 @@
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using QuantConnect.Api;
+using QuantConnect.Util;
+
+namespace QuantConnect.API
+{
+    /// <summary>
+    /// Node obj for API response
+    /// </summary>
+    public class Node
+    {
+        /// <summary>
+        /// The nodes cpu speed
+        /// </summary>
+        [JsonProperty(PropertyName = "speed")]
+        public decimal Speed { get; set; }
+
+        /// <summary>
+        /// The price of the node
+        /// </summary>
+        [JsonProperty(PropertyName = "price")]
+        public int Price { get; set; }
+
+        /// <summary>
+        /// CPU Count of node
+        /// </summary>
+        [JsonProperty(PropertyName = "cpu")]
+        public int CpuCount { get; set; }
+
+        /// <summary>
+        /// Size of RAM
+        /// </summary>
+        [JsonProperty(PropertyName = "ram")]
+        public decimal Ram { get; set; }
+
+        /// <summary>
+        /// Name of node
+        /// </summary>
+        [JsonProperty(PropertyName = "name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Node type (Sku)
+        /// </summary>
+        [JsonProperty(PropertyName = "sku")]
+        public string Sku { get; set; }
+
+        /// <summary>
+        /// String description of Node
+        /// </summary>
+        [JsonProperty(PropertyName = "description")]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Used by 
+        /// </summary>
+        [JsonProperty(PropertyName = "usedBy")]
+        public string UsedBy { get; set; }
+
+        /// <summary>
+        /// Project it is being used for
+        /// </summary>
+        [JsonProperty(PropertyName = "projectName")]
+        public string ProjectName { get; set; }
+
+        /// <summary>
+        /// Bool if the node is currently busy
+        /// </summary>
+        [JsonProperty(PropertyName = "busy")]
+        public bool Busy { get; set; }
+
+        /// <summary>
+        /// Full ID of node
+        /// </summary>
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Determine the appropriate SKU for a node by params
+        /// </summary>
+        /// <param name="cores">Number of cores</param>
+        /// <param name="memory">Size of RAM</param>
+        /// <param name="target">Target Environment Live/Backtest/Research</param>
+        /// <returns>Returns the SKU as a string for the node desired</returns>
+        public static string GetSKU(int cores, int memory, string target)
+        {
+            var result = $"{target[0]}";
+
+            if(cores == 0)
+            {
+                result += "-micro";
+            } 
+            else
+            {
+                result += cores + "-" + memory;
+            }
+
+            return result;
```

Comment:
To Upper

Reply:
Now that we use an enum for NodeTypes with the new SKU class  it will always be uppercase. The initial reason I didn't use ToUpper() was because of a warning message that said the behaviour of ToUpper could vary based on the users locale settings.

Reply:
You can use ToUpperInvariant(); it is still not upper here; `result += "-Micro";` -- 

Please also switch to use MS case; curious why didn't you use this:
[JsonProperty(PropertyName = "cores")]
public int Cores;


Then also `this` is redundant - it doesn't need to be mentioned; 
Cores = cores

Reply:
- SKU String is now always generated capitalized
- MS convention coding standards addressed in SKU class
- Dropped the `this`